.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - "10"
+cache:
+  yarn: true
+  directories:
+    - node_modules
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin::$(yarn global bin):$PATH
+script:
+  - yarn plugnet-dev-travis-build
```

Comment:
new requirement: Installing latest plugnet-dev in the Travis pipeline and using scripts from the installed latest plugnet-dev.